src/Tasks/Microsoft.NET.Build.Tasks/RuntimeOptions.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.NET.Build.Tasks
 {
     internal class RuntimeOptions
     {
+        public string tfm { get; set; }
```

Comment:
The property doesn't really align with the rest of the properties in the .json file.

1. Its casing is different than the existing properties.
2. It is an abbreviation, where the rest aren't.